.github/workflows/test.yml

```diff
@@ -12,11 +12,45 @@ on:
 jobs:
   test_github_action:
     runs-on: ubuntu-latest
+    continue-on-error: true
+    outputs:
+      output1: ${{ steps.test1.outcome }}
+      output2: ${{ steps.test2.outcome }}
 
     steps:
       - uses: actions/checkout@v2
-      - uses: char0n/apidom-validate@main
+
+      - id: test1
+        name: "Test 1: Validate at Severity level 1 (Errors)"
+        uses: amandalian/apidom-validate@main
+        with:
+          definition-file: ./.github/workflows/fixtures/openapi.json
+      - id: test2
+        name: "Test 2: Validate at Severity level 4 (Hints)"
+        if: success() || failure()
```

Comment:
quick question -  if `cancelled()` we want to stop this step right?

Reply:
yes that was my intention!